docs/concepts/spsp.md

```diff
@@ -0,0 +1,21 @@
+## SPSP
+
+[SPSP](https://github.com/interledgerjs/ilp-protocol-spsp) (Simple Payment Setup Protocol) is an 
+[application layer protocol](https://github.com/interledger/rfcs/blob/master/0001-interledger-architecture/0001-interledger-architecture.md#protocol-layers), 
+which uses HTTPS endpoints to securely exchange payment details, and then allows the user to send and receive value 
+over Interledger. 
+
+SPSP provides lightweight client/server APIs to send and receive value. An SPSP client sends (pay) value, and an 
+SPSP server receives value. If *pull* value is supported, a server can send value and a client can receive (pull) value. 
+
+When you start an SPSP server, it:
+* Connects to a `moneyd` instance
+* Creates a receiver
+* Opens a port to listen to
+* Generates a [payment pointer](https://github.com/interledger/rfcs/blob/master/0026-payment-pointers/0026-payment-pointers.md)<br/>
```

Comment:
I'm not sure if this is the right place to describe payment pointers; maybe that could be broken into a separate paragraph. The SPSP server doesn't really generate a payment pointer so much as a payment pointer can be used to nicely format the URL of an SPSP server